app/views/layouts/_communicart_header.html.erb

```diff
@@ -32,6 +32,8 @@
   </div>
 </div>
 
-<ul id='breadcrumb-nav'>
-  <li><%= link_to "Back to main portal", carts_path %></li>
-</ul>
+<%- unless excluded_portal_link %>
```

Comment:
I still want to exclude the home page so would need to keep this within the excluded_portal_link method